modules/gwlb_endpoint_set/main.tf

```diff
@@ -28,6 +28,16 @@ locals {
   input_routes = { for v in local.input_routes_flat : "${v.routekey}-${v.subnetkey}" => v }
 }
 
+resource "time_sleep" "this" {
+  count = var.delay ? 1 : 0
+
+  create_duration = "60s"
```

Comment:
Is there any reason to have constant `60s` value ?
Maybe it's better to store it in variable ?
Or maybe instead of having `delay` as bool , let's make it number and create that resource, if number is > 0 ?